instruction/db-sql/db-sql.md

```diff
@@ -88,20 +91,28 @@ Notice that each field is followed by the `NOT NULL` clause. That means each of
 
 The `id` field is also annotated with the `AUTO_INCREMENT` keyword. This means that you don't actually provide the `id` field when you insert a row. The database will do that for you using an automatically increasing integer.
 
+### Changing Tables
+
+> [!IMPORTANT]
+> Simply changing a "Create Table" statement stored in the code **does not** change the database. Read on to understand valid ways to change a SQL table.
+
+A `CREATE TABLE` statement _only_ does the action implicit in its name. If the table already exists (a common scenario), this command will throw an error; its sister, `CREATE TABLE IF NOT EXISTS`, will suppress the error and perform a no-op operation on the database.
```

Comment:
no-op sound like jargon to me. Why not just say "doesn't perform the create operation on the database"?